gem2rpm/nonscl.spec.erb

```diff
@@ -39,6 +40,12 @@ BuildRequires: rubygems-devel <%= req %>
 <% end -%>
 <% if spec.extensions.empty? -%>
 BuildArch: noarch
+<% else -%>
+<% for d in spec.runtime_dependencies -%>
+<% for req in d.requirement -%>
+Requires: rubygem(<%= d.name %>) <%= req  %>
```

Comment:
I probably messed it up and used Requires here instead of BuildRequires, but I saw pinning it to `%check` somewhere else:
```suggestion
BuildRequires(check): rubygem(<%= d.name %>) <%= req  %>
```

Reply:
```console
$  rpm -q --specfile /home/evgeni/Devel/theforeman/foreman-packaging/packages/foreman/rubygem-pg/rubygem-pg.spec
error: line 33: Unknown tag: BuildRequires(check): rubygem(bigdecimal)
```

No? ;)

But I agree that this should be `BuildRequires` not `Requires`

Reply:
825d56cbb75b70ad4dad6ec1892b10da81add111 dropped it and it was hidden behind `if 0` so perhaps it used to work but is no longer supported? Either way, I thought it was a nice idea but clearly doesn't work :)

Reply:
idk, never seen that syntax before. either way, it's good enough now :)